invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/previewer/iiif_simple.less

```diff
@@ -0,0 +1,16 @@
+/*
+ *   Copyright (C) 2022 data-futures.
+ *
+ * Invenio App RDM is free software; you can redistribute it and/or modify
+ * it under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+.iiif-simple-preview {
+    display: flex;
+    justify-content: center;
+}
+
+.iiif-simple-preview img {
+    max-width: 100%
+}
+
```

Comment:
this part should be implemented in `site.overrides`, then no css bundle is needed anymore (due to centralised theme)